client/styles/_reset.scss

```diff
@@ -0,0 +1,48 @@
+* {
```

Comment:
@AStashevskaya Lets try to use https://github.com/necolas/normalize.css as module instead of this file